client/my-sites/site-settings/settings-newsletter/BylineSettings.tsx

```diff
@@ -0,0 +1,101 @@
+import { Gravatar, FormLabel } from '@automattic/components';
+import { ToggleControl, Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
+import {
+	getLocalizedDate,
+	phpToMomentDatetimeFormat,
+} from 'calypso/my-sites/site-settings/date-time-format/utils';
+import { useSelector } from 'calypso/state';
+import { getCurrentUser } from 'calypso/state/current-user/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import { BylinePreview } from './BylinePreview';
+
+const GRAVATER_OPTION = 'jetpack_gravatar_in_email';
+const AUTHOR_OPTION = 'jetpack_author_in_email';
+const POST_DATE_OPTION = 'jetpack_post_date_in_email';
+
+type BylineSettingsProps = {
+	showAvatarValue?: boolean;
+	showAuthorValue?: boolean;
+	showDateValue?: boolean;
+	handleToggle: ( field: string ) => ( value: boolean ) => void;
+	disabled?: boolean;
+	dateFormat?: string;
+};
+
+export const BylineSettings = ( {
+	showAvatarValue,
+	showAuthorValue,
+	showDateValue,
+	handleToggle,
+	disabled,
+	dateFormat = 'F j, Y',
+}: BylineSettingsProps ) => {
+	const translate = useTranslate();
+	const site = useSelector( getSelectedSite );
+	const user = useSelector( getCurrentUser );
+	const siteSlug = site?.slug || '';
+
+	const localizedDate = getLocalizedDate( dateFormat );
+
+	return (
+		<>
+			<FormLabel className="increase-margin-bottom-fix">
+				{ translate( 'For each new post email, include' ) }
```

Comment:
Since this is a partial string, can you add a comment explaining it for translators?